libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py

```diff
@@ -22,6 +22,8 @@
     'hwdef', type=str, default=None, help='hardware definition file')
 parser.add_argument(
     '--params', type=str, default=None, help='user default params path')
+parser.add_argument(
```

Comment:
using nargs='+' for the hwdef argument would be neater I think, so args.hwdef becomes a list

Reply:
wscript doesn't seem to accept nargs, which I assume is the reason the configure doesn't work when I changed to nargs. Any thoughts?

Reply:
see comment below